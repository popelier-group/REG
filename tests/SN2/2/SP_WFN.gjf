# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -0.40951200   -0.00005100    0.00007400
 H                 -0.71913300   -0.87927400   -0.54721000
 H                 -0.71918000    0.91350300   -0.48771900
 H                 -0.71911500   -0.03439900    1.03515200
 Br                 1.62564600   -0.00000600    0.00003100
 Cl                -3.46608000    0.00003700   -0.00010300

frame.wfn
