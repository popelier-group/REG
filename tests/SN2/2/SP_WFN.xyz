
6

C       -0.298546    0.004215    0.035304
H       -0.510504   -0.540177    0.944770
H       -0.719923   -0.477787   -0.835634
H       -0.587603    1.043401    0.105712
Br        1.722372   -0.023857   -0.203379
Cl       -3.333749    0.046133    0.393623
