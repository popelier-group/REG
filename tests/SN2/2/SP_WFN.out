 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.2/Gau-75177.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.2/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     75178.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -0.40951  -0.00005   0.00007 
 H                    -0.71913  -0.87927  -0.54721 
 H                    -0.71918   0.9135   -0.48772 
 H                    -0.71912  -0.0344    1.03515 
 Br                    1.62565  -0.00001   0.00003 
 Cl                   -3.46608   0.00004  -0.0001 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.409512   -0.000051    0.000074
      2          1           0       -0.719133   -0.879274   -0.547210
      3          1           0       -0.719180    0.913503   -0.487719
      4          1           0       -0.719115   -0.034399    1.035152
      5         35           0        1.625646   -0.000006    0.000031
      6         17           0       -3.466080    0.000037   -0.000103
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.080934   0.000000
     3  H    1.080933   1.793764   0.000000
     4  H    1.080935   1.793790   1.793782   0.000000
     5  Br   2.035158   2.563313   2.563320   2.563311   0.000000
     6  Cl   3.056568   2.935682   2.935583   2.935772   5.091726
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -0.298546    0.004215    0.035304
      2          1           0       -0.510504   -0.540177    0.944770
      3          1           0       -0.719923   -0.477787   -0.835634
      4          1           0       -0.587603    1.043401    0.105712
      5         35           0        1.722372   -0.023857   -0.203379
      6         17           0       -3.333749    0.046133    0.393623
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         155.8455141           0.7973006           0.7973006
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       174.6670422494 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  9.54D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 EnCoef did   100 forward-backward iterations
 SCF Done:  E(RB3LYP) =  -3074.39483035     A.U. after   12 cycles
            NFock= 12  Conv=0.12D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.69025-101.25694 -62.34325 -56.16312 -56.15991
 Alpha  occ. eigenvalues --  -56.15991 -10.08236  -9.16994  -8.55261  -6.92859
 Alpha  occ. eigenvalues --   -6.92805  -6.92805  -6.38243  -6.37184  -6.37184
 Alpha  occ. eigenvalues --   -2.49357  -2.49057  -2.49057  -2.48218  -2.48218
 Alpha  occ. eigenvalues --   -0.66306  -0.54023  -0.50905  -0.30248  -0.30248
 Alpha  occ. eigenvalues --   -0.24417  -0.15802  -0.15802  -0.05789  -0.05639
 Alpha  occ. eigenvalues --   -0.05639
 Alpha virt. eigenvalues --    0.09642   0.10828   0.13976   0.13977   0.16018
 Alpha virt. eigenvalues --    0.17386   0.17988   0.17989   0.21510   0.21823
 Alpha virt. eigenvalues --    0.21823   0.24306   0.24306   0.24526   0.30856
 Alpha virt. eigenvalues --    0.30856   0.31485   0.33994   0.40465   0.50139
 Alpha virt. eigenvalues --    0.52133   0.52133   0.53879   0.53879   0.65988
 Alpha virt. eigenvalues --    0.67792   0.67792   0.70586   0.74577   0.74577
 Alpha virt. eigenvalues --    0.75590   0.77067   0.77067   0.82557   0.82557
 Alpha virt. eigenvalues --    0.84123   0.95370   1.07776   1.14926   1.14926
 Alpha virt. eigenvalues --    1.15312   1.15312   1.16499   1.22782   1.22783
 Alpha virt. eigenvalues --    1.23009   1.56452   1.67290   1.67291   1.80363
 Alpha virt. eigenvalues --    1.84334   1.84334   2.08359   2.12326   2.12327
 Alpha virt. eigenvalues --    2.28696   2.30913   2.30914   2.52084   2.52085
 Alpha virt. eigenvalues --    2.55382   2.59729   2.64378   2.64379   2.68007
 Alpha virt. eigenvalues --    2.68008   3.10796   3.13730   3.85601   3.85602
 Alpha virt. eigenvalues --    4.27675   4.27675   4.36894   4.36894   4.65566
 Alpha virt. eigenvalues --    6.72929   7.69081   7.69082   7.79322   9.87150
 Alpha virt. eigenvalues --   24.03956  26.07885  26.07886  26.18671  48.02294
 Alpha virt. eigenvalues --  215.83516 289.89048 289.89048 290.011201020.80026
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.057847   0.409807   0.409815   0.409798   0.100262  -0.047403
     2  H    0.409807   0.527480  -0.035887  -0.035878  -0.053925   0.009162
     3  H    0.409815  -0.035887   0.527464  -0.035881  -0.053922   0.009162
     4  H    0.409798  -0.035878  -0.035881   0.527495  -0.053927   0.009162
     5  Br   0.100262  -0.053925  -0.053922  -0.053927  35.353206   0.007637
     6  Cl  -0.047403   0.009162   0.009162   0.009162   0.007637  17.910540
 Mulliken charges:
               1
     1  C   -0.340126
     2  H    0.179240
     3  H    0.179249
     4  H    0.179230
     5  Br  -0.299331
     6  Cl  -0.898261
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C    0.197593
     5  Br  -0.299331
     6  Cl  -0.898261
 Electronic spatial extent (au):  <R**2>=           1216.4467
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=             11.0416    Y=             -0.1527    Z=             -1.3036  Tot=             11.1194
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -98.2613   YY=            -43.8183   ZZ=            -44.5671
   XY=              0.7536   XZ=              6.4294   YZ=             -0.0890
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -36.0457   YY=             18.3973   ZZ=             17.6485
   XY=              0.7536   XZ=              6.4294   YZ=             -0.0890
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=            237.5015  YYY=             -0.1086  ZZZ=             -8.7024  XYY=             23.6063
  XXY=             -2.6469  XXZ=            -22.4852  XZZ=             25.8559  YZZ=             -1.2165
  YYZ=             -2.6956  XYZ=              0.1725
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=          -2093.5243 YYYY=            -70.3988 ZZZZ=            -91.2129 XXXY=             17.6236
 XXXZ=            150.1960 YYYX=              9.9156 YYYZ=             -1.1686 ZZZX=             89.6201
 ZZZY=             -1.4109 XXYY=           -274.0152 XXZZ=           -284.5676 YYZZ=            -26.9223
 XXYZ=             -1.1486 YYXZ=             29.5351 ZZXY=              4.0343
 N-N= 1.746670422494D+02 E-N=-7.700566530365D+03  KE= 3.069988349832D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE366\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\ROOT\11-Nov-20
 20\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\C,0,
 -0.409512,-0.000051,0.000074\H,0,-0.719133,-0.879274,-0.54721\H,0,-0.7
 1918,0.913503,-0.487719\H,0,-0.719115,-0.034399,1.035152\Br,0,1.625646
 ,-0.000006,0.000031\Cl,0,-3.46608,0.000037,-0.000103\\Version=ES64L-G1
 6RevC.01\State=1-A\HF=-3074.3948303\RMSD=1.228e-09\Dipole=4.3746959,-0
 .000078,0.000177\Quadrupole=-27.3712805,13.6856137,13.6856668,0.000453
 1,-0.0012431,0.0000048\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 SCIENCE AND PEACE WILL TRIUMPH OVER IGNORANCE AND WAR

                          -- PASTEUR
 Job cpu time:       0 days  0 hours  0 minutes 30.6 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 16.5 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 18:34:09 2020.
